UiTests/AnyOrgOrPersonalUiTest/AnyOrgOrPersonalUiTest.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  
+  <PropertyGroup>
+    <TargetFrameworks>net8.0</TargetFrameworks>
+    <!--<TargetFrameworks Condition="'$(TargetNet9)'== 'True'">$(TargetFrameworks); net9.0</TargetFrameworks>-->
```

Comment:
If this will only conditionally target .NET 9, why leave it in a comment?

Reply:
Valid, this is from the Id Web repo where not all .NET 9 testing was wanting to include UI tests, which as you point out doesn't matter here. I'll uncomment and add the field to the props file.